dnSpy/dnSpy/Search/FrameworkFileUtils.cs

```diff
@@ -457,7 +457,7 @@ static class FrameworkFileUtils {
 
 		public static bool IsFrameworkAssembly(string filename, string? assemblySimpleName) {
 			// Check if it's in one of the .NET runtime dirs
-			if (Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(filename))) is string baseDir && Directory.Exists(Path.Combine(baseDir, "Microsoft.NETCore.App")))
+			if (String.IsNullOrWhiteSpace(filename) == false && Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(filename))) is string baseDir && Directory.Exists(Path.Combine(baseDir, "Microsoft.NETCore.App")))
```

Comment:
Please use the `string2` instead of `String` since the one in the core library does not have nullability annotations for .NET Framework.
Furthermore, consider using the `!` negation operator instead of comparing the result with `false`, this would be more consistent with the current coding practices applied in the dnSpy code base.